KEYRUPT_UPRUPT.s

```diff
@@ -127,7 +127,7 @@ ELRCODE		OCT	22
 # THE RECEPTION OF A BAD CODE (I.E., CCC FAILURE) LOCKS OUT FURTHER UPLINK ACTIVITY BY SETTING BIT4 OF FLAGWRD7 = 1.
 # THIS INDICATION WILL BE TRANSFERRED TO THE GROUND BY THE DOWNLINK WHICH DOWNLINKS ALL FLAGWORDS.
 # WHEN UPLINK ACTIVITY IS LOCKED OUT, IT CAN BE ALLOWED WHEN THE GROUND UPLINKS AND `ERROR RESET' CODE.
-# (IT IS RECOMMENDED THAT THE `ERROR LIGHT RESET' CODE IS PRECEEDED BY 16 BITS THE FIRST OF WHICH IS 1 FOLLOWED
+# (IT IS RECOMMENDED THAT THE `ERROR LIGHT RESET' CODE IS PRECEDED BY 16 BITS THE FIRST OF WHICH IS 1 FOLLOWED
```

Comment:
This typo is present in the scan of [page 1451](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1451.jpg).